pyxdsm/XDSM.py

```diff
@@ -126,14 +127,18 @@ def __init__(self, use_sfmath=True, optional_latex_packages=None):
             Whether to use the sfmath latex package, by default True
         optional_latex_packages : string or list of strings, optional
             Additional latex packages to use when creating the pdf and tex versions of the diagram, by default None
+        auto_fade : dictionary, optional
+            For each key "inputs", "outputs", "connections", and "processes", the value can be one of:
+            - "all" : fade all blocks
+            - "unconnected" : fade all components connected to faded blocks
```

Comment:
Would it not make more sense to call this option "connected", since it fades everything connected to the block?

Reply:
Yeah you're right I confused myself with the double negative. I've pushed a fix.